pvlib/modelchain.py

```diff
@@ -754,10 +752,6 @@ def prepare_inputs(self, times=None, irradiance=None, weather=None):
                    "Please use the weather parameter to pass a DataFrame " +
                    "with irradiance (ghi, dni, dhi), wind speed and " +
                    "temp_air.\n")
-        if irradiance is not None:
```

Comment:
The lines from "# The following part could be removed together with the irradiance" through "# **** End ****" should also be deleted.